### DISCUSSION TITLE
Gravity Wave tuning

### DISCUSSION THREAD

**Discussion by JulioTBacmeister** [NCAR/amwg_dev#248] [TF2: Tuning]
Centralized location to collect input on tuning gravity waves in CAM7 using both L58 and L93 configurations.

**Comment by dan800** [comment]
@cecilehannay @JulioTBacmeister Perhaps this is the place to discuss the differences between the MT and LT namelists. 
For the beginning MT configuration, I would start with gravity wave settings that are the same as WACCM-SE 1˚. I am not sure what those are - perhaps Nick Davis has a favorite current configuration? We would want to set   

tau_0_ubc              = .true.

Regarding the flux condition for constituents, I would go with zero flux to begin with for everything.

**Comment by nadavis-ncar** [comment]
I can only comment on ne16 with any amount of certainty. I'll have an answer on ne30 in about a week, I just submitted some initial runs.

**Reply by dan800** [reply]
Should we just go with what comes out of the box with WACCM for now?

**Reply by nadavis-ncar** [reply]
Its not dysfunctional, maybe just not optimized in the MLT.

**Reply by dan800** [reply]
To paraphrase @andrewgettelman,  "all models suck, we just aim to suck the least"

**Comment by dan800** [comment]
Discussion point: 

Should **all** CAM7 configurations have as a default the following namelist settings.

<use_gw_convect_dp> .true. </use_gw_convect_dp>
<use_gw_front> .true. </use_gw_front>

The exact parameter settings may differ across the LT and MT configurations _but the same physics is operational in both versions_. 

In particular - should both models have:

<tau_0_ubc> .true. </tau_0_ubc>

Should the convection source be reduced in LT to slow down the QAO/QBO?

This follows on from https://github.com/NCAR/amwg_dev/discussions/261#discussioncomment-5892584

Finally, to make an informed decision, it would be good to decide on key metrics and have the ability to compare to obs. (ERA5?) For example, NH surface pressure in DJF, TEM circulation, frequency of SSWs, timing of the spring time break up of the SH polar vortex. 

@JulioTBacmeister @nadavis-ncar @islasimpson, @tilmes, @adamrher,  Doug,  Rolando, please comment.

**Reply by JulioTBacmeister** [reply]
As soon as the coupled evaluation starts we're going to run L58 experiments:

1)  https://github.com/NCAR/amwg_dev/issues/265
use_gw_front=.true
use_gw_convect_dp=.true
tau_0_ubc=.true

2) #267 
use_gw_front=.true
use_gw_convect_dp=.true
tau_0_ubc=.false.

**Reply by dan800** [reply]
new basline: 
f.cam6_3_109.FLTHIST_v0b.ne30.tuningF.001 **#259**
No non-orographic GW 

f.cam6_3_109.FLTHIST_v0b.ne30.tuningF_non-ogw.001 **#269** 
Same as https://github.com/NCAR/amwg_dev/issues/259 + non-orographic GW tau_0_ubc=.true

NH Polar surface pressure vs. observed. 

**259**
![image](https://github.com/NCAR/amwg_dev/assets/37961632/04a9a548-583c-4da4-8aee-f2d08d5fa093)

 **269**
![image](https://github.com/NCAR/amwg_dev/assets/37961632/f20c5f1b-afe2-4a2f-a60f-909ac21a8e3e)

**Comment by dan800** [comment]
U DJF
![image](https://github.com/NCAR/amwg_dev/assets/37961632/dd2f66a4-ea3f-4712-a552-4276bb4b3ec8)

![image](https://github.com/NCAR/amwg_dev/assets/37961632/54926fd2-60f0-416d-8bff-9025e9a7028b)

**Comment by dan800** [comment]
U JJA
![image](https://github.com/NCAR/amwg_dev/assets/37961632/55a42151-56ae-4162-b842-151738b21b95)
![image](https://github.com/NCAR/amwg_dev/assets/37961632/762cf76e-250d-4633-80eb-03cd0ec89b35)

**Comment by dan800** [comment]
![image](https://github.com/NCAR/amwg_dev/assets/37961632/bc8e7b8c-a684-49f2-8431-d374a9b25ead)
![image](https://github.com/NCAR/amwg_dev/assets/37961632/dc982e53-5d6f-4fb7-bd2b-b8122c2ed04a)

**Comment by dan800** [comment]
![image](https://github.com/NCAR/amwg_dev/assets/37961632/ca4b3cdb-d361-429f-a79c-c22f476ad57e)
![image](https://github.com/NCAR/amwg_dev/assets/37961632/6f0e5ffe-dda5-4ae4-b731-cbff995120f2)

**Reply by nadavis-ncar** [reply]
Here's a comparison that includes the L135's current "best" option, f.e23.FWmaHIST.ne16pg3_mg17_L135_ro3_r004.

DJF:
![zonal_wind_djf_v2(1)](https://github.com/NCAR/amwg_dev/assets/63112962/c4cdfc0a-8efb-49dc-a40b-392c51a4b404)

JJA:
![zonal_wind_jja_v2(1)](https://github.com/NCAR/amwg_dev/assets/63112962/20c5055c-3809-4b2e-8c9e-44495d7e4699)

**Reply by nadavis-ncar** [reply]
Colorbar was cut off, apologies. Mean is every 10 m/s and difference is every 3 m/s.


DJF:
![zonal_wind_djf_v2(2)](https://github.com/NCAR/amwg_dev/assets/63112962/e9ef71cc-aa62-49c3-b95f-6ea3fe0a3282)

JJA:
![zonal_wind_jja_v2(2)](https://github.com/NCAR/amwg_dev/assets/63112962/dfb83274-733a-4b37-8317-b84df7da1f4a)

**Comment by JulioTBacmeister** [comment]
Seasonal-Zonal means of U from #268 and #260
#268 has modified ridge settings that attempt to put more of the drag into propagating waves vs low-level obstacle drag:

effgw_rdg_beta = 2.0D0
effgw_rdg_beta_max = 2.0D0
rdg_beta_cd_llb = 0.25D0

==>11-years long

#260 is the control
==> 9 years long

DJF
![image](https://github.com/NCAR/amwg_dev/assets/9221710/264eb1a5-39d0-4c16-a782-d4c1a5dc7c28)

JJA
![image](https://github.com/NCAR/amwg_dev/assets/9221710/fce13c13-cb82-4c6d-b8a4-7c09ff125179)

**Reply by JulioTBacmeister** [reply]
Looks like this ridge tuning change has the desired effect on zonal winds especially in DJF

**Reply by dan800** [reply]
@Julio So is this ridge tuning something that should be done also in WACCM & MT? i.e., it's scientifically defensible and represents missing or improved physics? If so, we should see what it does in HT/MT that also have non-orographic GWs turned on. We want to have as few differences in operating physical processes between all the model configurations. This would give us more confidence we are getting a better answer for the right reasons.

**Reply by JulioTBacmeister** [reply]
@dan800 Yes. I think we should try this tuning in the MT and WACCM configurations, as well as in LT with the non-orographic waves also on.  The biases in the run above rare educed but not eliminated. I'm curious to see what happens with the non-orographic contributing also.

**Reply by tilmes** [reply]
@JulioTBacmeister I think, Cecile said she would run it if we do a run request for the MT version. It would be great if we can start those asap!

**Reply by dan800** [reply]
> @dan800 Yes. I think we should try this tuning in the MT and WACCM configurations, as well as in LT with the non-orographic waves also on. The biases in the run above rare educed but not eliminated. I'm curious to see what happens with the non-orographic contributing also.

And seasonal means hide things like the seasonal cycle. Can you tell me what the UBC condition is for oro-graphic waves? Looking at where the zero wind lines are, these waves could easily reach the lid (DJF >40N).

PS looks better in this run than #259 
![image](https://github.com/NCAR/amwg_dev/assets/37961632/6218906e-540b-4f41-bff9-9e7ee1fa2e00)

**Reply by nadavis-ncar** [reply]
@JulioTBacmeister Is there a reason to set rdg_beta_cd_llb = 0.25 regardless of rdg_beta, or is the ratio between rdg_beta and rdg_beta_cd_llb important?

**Comment by dan800** [comment]
#271 non-oro with tau_0_ubc = .false. does not give the same improvement in DJF PS as #269 
It looks almost like #259
 
![image](https://github.com/NCAR/amwg_dev/assets/37961632/511c2789-3a77-4135-96b8-34bdd892c458)

**Comment by islasimpson** [comment]
[Here](https://docs.google.com/presentation/d/1n2lJG0mhPKjGhvRrLEVUl4aQvjk36gpdyZytLjwPhos/edit?usp=sharing) are some plots of the latest runs.  I agree that tau_0_ubc=T looks better in the LT at this point.  When tau_0_ubc = F the NH vortex is too strong and there are associated biases in the North Atlantic jet stream in the tropics.  For the SH, it's a bit less clear to me.  The vortex is too strong and breaks down too late in all cases, including MT.  

Some issues that I think we have with tau_0_ubc=T in the LT are these zonal wind structures in the tropics that seem to arise because the oscillation is phase locked to the seasonal cycle.  The NH vortex is also too weak and this is true in the MT run as well. 

Some other things to note (unrelated to GW tuning) are that I think our asymmetries in the SH jet stream in DJF have degraded compared to CAM6 and there seems to be a pretty big bias in the jet stream in East Asia and the North Pacific during the summer.

I wonder if we should start outputting daily TEM diagnostics to allow us to look at some of the daily timescale variability in the vortex?

**Reply by JulioTBacmeister** [reply]
Thanks Isla. I am mentioning @swrneale @dan800 @brianpm @cecilehannay @PeterHjortLauritzen @tilmes @adamrher  just to start with.  I think everyone will be interested in Isla's results linked above.  

I believe many of the issues identified have been with us since the early days of L58.  Look through the diagnostics for issue #13.  There is an interesting shift in tropical Pacific precipitation between the L58 and L32 control.

**Reply by islasimpson** [reply]
Added some water vapor plots in slides 15 to 17 [here](https://docs.google.com/presentation/d/1n2lJG0mhPKjGhvRrLEVUl4aQvjk36gpdyZytLjwPhos/edit?usp=sharing).  The stratosphere seems very moist.  Pasting in the zonal mean Q below.  @tilmes - could there be something wrong with the chemical generation of water vapor?

![Screenshot from 2023-05-24 13-57-43](https://github.com/NCAR/amwg_dev/assets/41842751/ad7712a4-81e9-4747-a407-68cb09a2f466)

**Reply by dan800** [reply]
@isla I think these have the duplicate water vapor source in the stratosphere - from chemistry (methane oxidation) and the prescribed water input from a file. This has been fixed. Though IIRC we should see numbers on the order of 6 ppmv in the upper stratosphere/lower mesosphere, so it will blow out you scale. BTW is the volume or mass mixing ratio?

**Reply by dan800** [reply]
@cecilehannay @islasimpson @JulioTBacmeister @tilmes I have submitted a run request to see if removing the duplicate source dries the stratosphere (#276)

**Reply by dan800** [reply]
> Some issues that I think we have with tau_0_ubc=T in the LT are these zonal wind structures in the tropics that seem to arise because the oscillation is phase locked to the seasonal cycle.

It would be good to start a few runs based on #276 with reduced convective non-orographic gravity wave forcing - what parameter in that source has been proven in the past to be effective in tuning the period of the QBO?

**Reply by dan800** [reply]
#279 vs #282

![image](https://github.com/NCAR/amwg_dev/assets/37961632/fa053016-4a4c-4367-9158-577841987a64)

![image](https://github.com/NCAR/amwg_dev/assets/37961632/9a92ed33-305e-4b95-9e41-a741f6dfaa79)

**Comment by JulioTBacmeister** [comment]
Hi Isla,
All the panels have #263 as the title. Is that right?

On Wed, May 24, 2023 at 2:01 PM islasimpson ***@***.***>
wrote:

> Added some water vapor plots in slides 15 to 17 here
> <https://docs.google.com/presentation/d/1n2lJG0mhPKjGhvRrLEVUl4aQvjk36gpdyZytLjwPhos/edit?usp=sharing>.
> The stratosphere seems very moist. Pasting in the zonal mean Q below.
> @tilmes <https://github.com/tilmes> - could there be something wrong with
> the chemical generation of water vapor?
>
> [image: Screenshot from 2023-05-24 13-57-43]
> <https://user-images.githubusercontent.com/41842751/240728540-ad7712a4-81e9-4747-a407-68cb09a2f466.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/amwg_dev/discussions/248#discussioncomment-5994025>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACGLMTSW4P6X5H5NAHP2YOLXHZSKBANCNFSM6AAAAAAW6W7GRU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*My working day may not be your working day. Please do not feel obliged to
reply to this message outside of your normal working hours.*

**Reply by islasimpson** [reply]
Oops, no.
![Screenshot from 2023-05-24 14-09-29](https://github.com/NCAR/amwg_dev/assets/41842751/d5f33bc1-4b6d-440b-96f4-a4867048c40d)

**Reply by tilmes** [reply]
@adamrher didn't we see changes in the stratospheric water vapor when we did the different tunings for the model earlier?
We have done the same chemistry for earlier specified chemistry runs with the 93L model and WACCM SC, and usually  the stratosphere was dry. So, I don't think that is any  way the chemistry could to it.

**Reply by adamrher** [reply]
The tuning should only increase UTLS values by ~50%; what we see here is much larger. I suspect the top b.c.'s of water vapor are related to this; and so we need to see what this new run that Dan requested looks like: https://github.com/NCAR/amwg_dev/issues/276

**Reply by dan800** [reply]
@adamrher This is not a b.c. issue but a double counting by specifying an in situ source (ext_frc_specifier for H2O) within the model domain that injects water when the same source (CH4 -> 2*H2O) is included in the ghg_mam4 chemistry scheme. Let's move discussion of this to Julio's new thread on water vapor.

**Comment by JulioTBacmeister** [comment]
![image](https://github.com/NCAR/amwg_dev/assets/9221710/1fbc7944-d165-4fec-a138-2434899fb8f7)

Well, this may not be the best result we could have expected, but it is promising that the secondary annual westerly maxima e.g. 1995, 1997 aren't descending as far as the primary maxima.

**Reply by dan800** [reply]
Well we always knew the LT would be compromised for stratospheric dynamics. We could fire off another run with even lower effgw_beres_dp, but frankly this is another reason to drop tuning up a model that is not the workhorse model and move on to the MT configuration for further tuning. @rgac1a might want to comment. On the bright side, at least in the multi-year mean, this setting will reduce the (ERA5 - LT) differences, since the mean is less westerly.

**Reply by rgarc1a** [reply]
I take it the middle panel above has effgw_beres = 0.3 and the RHS panel has effgw_beres = 0.5. Both look bad: apparent annual phase locking in both; westerlies stronger than easterlies in 0.5; vanishing QBO amplitude in 0.3; and so on. A surprise is that reducing effgw_beres did not change the period appreciably but instead reduced the amplitude. With such a low top, I presume at least half of the stratosphere is in the sponge layer; thus, it does not seem likely we will get a realistic QBO in this model. However, given enough computing resources and high tolerance for frustration, one could try one more thing: NOT dumping ONLY the Beres parameterization momentum at the top. Equatorial dynamics does not work like extratropical dynamics and, in the QBO regime, one expects direct acceleration of the zonal wind to dominate secondary circulation effects; thus, the reason why we dump momentum elsewhere does not apply to the deep Tropics. 

BTW, as I recall from presentations at the co-chairs meetings, LT runs with tau_0_ubc = F do not get stuck with in annual tropical wind oscillation but I don't believe I have seen a plot of U(z,t) at the Equator for any of those runs. We should have a look at those first and see if they have anything like a QBO. If this is not the caee, it may be time to move on.

**Reply by dan800** [reply]
> ... one could try one more thing: NOT dumping ONLY the Beres parameterization momentum at the top. 

I would think this requires a code change. @fvitt would know best.

>  ... LT runs with tau_0_ubc = F do not get stuck with in annual tropical wind oscillation but I don't believe I have seen a plot of U(z,t) at the Equator for any of those runs. 

IIRC @islasimpson showed a comparison - there is practically no tropical oscillation at all in the run with tau_0_ubc = .false.

As an example, here is run #271 which has tau_0_ubc = .false.

![image](https://github.com/NCAR/amwg_dev/assets/37961632/d9677d54-5a5d-4500-a8ed-89b8b0caf966)
Full diags here: https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_109.FLTHIST_v0b.ne30.tuningF_non-ogw_ubcF.001/atm/f.cam6_3_109.FLTHIST_v0b.ne30.tuningF_non-ogw_ubcF.001_1995_2006_vs_Obs/

**Reply by rgarc1a** [reply]
I missed that (Equatorial U(z,t) for no GW momentum dumping). Looks like it's time to move on, then.

**Comment by rgarc1a** [comment]
Well, that is not encouraging, so I wouldn’t bother with setting tau_0_ubc=false only for the Beres GW. It ain’t gonna work. I presume this #259 run had effgw_beres - 0.5?


Rolando Garcia
NCAR/ACOM
P.O. Box 3000
Boulder, CO 80307-3000
(303) 497-1446
***@***.***

**Reply by dan800** [reply]
Actually I just realized that #271 is the one to look at (non-oro on and tau_0_ubc false) rather than #259. I updated the figure above.

Namelist is on GitHub:
https://github.com/NCAR/amwg_dev/blob/f.cam6_3_109.FLTHIST_v0b.ne30.tuningF_non-ogw_ubcF.001/CaseDocs/atm_in
It does have effgw_beres_dp		= 0.55D0

**Comment by rgarc1a** [comment]
effgw_beres is set to 0.5 on that one (#271)too, if I read the namelist correctly. This setting has been enough to produce a QBO in higher top models (WACCM, CAM MT). In #271 there appears to be (per the updated plot above) an annual oscillation with mostly easterly wind maxima (and very weak or nonexistent westerlies). This is not any better than #259 and I don't see much point on persevering with it.

**Comment by JulioTBacmeister** [comment]
@dan800 @nadavis-ncar @cecilehannay @rgarc1a @PeterHjortLauritzen @islasimpson:

Simply raising taubgnd 2.5D-3 from 1.25D-3  has some positive effects but also some very bad ones in FMT

The cold bias in the SH stratosphere gets a little better, but there is extreme warming further up. You can see the beginnings of it in the following.
![image](https://github.com/NCAR/amwg_dev/assets/9221710/c6e431f6-9e08-4f6f-b529-4db12bcc6a39)
![image](https://github.com/NCAR/amwg_dev/assets/9221710/5aa18e3b-f1a2-4cad-bae4-8bebee47d33e)

Zonal wind improves.
![image](https://github.com/NCAR/amwg_dev/assets/9221710/c831825e-e685-45e0-aa2b-0e27f075706d)

Full depth of FMT showing extremely high T in mesosphere (note color scale).
![image](https://github.com/NCAR/amwg_dev/assets/9221710/1432ab99-0196-4926-8355-983b4fc5b145)

Maybe playing with the width of spectrum can help here.

**Reply by rgarc1a** [reply]
The experience with FV WACCM6 was that increasing taubgnd for the frontal spectrum makes the waves break at lower altitudes, reduces overall zonal-mean vertical velocity near the mesopause and thus increases mesopause T. I am not sure playing with the spectrum is going to do much good. What we found most effective to control lower polar stratosphere T in FV WACCM was the settings of the orographic GW parameterization. Whatever those settings were in WACCM6 produced acceptable T as documented in Gettelman et al. (2019), Fig. 3. The main problem with that model was that the remnant of the polar vortex in the lower stratosphere persisted too long and led to a sizable cold bias in Oct-Dec. We could live with that, as T during the development of the O3 hole (Sept-early Oct) was OK in the lower stratosphere (Fig. 9A of G et al.) I wonder why we cannot reproduce the sam behavior in WACCM SE.
